src/main/scala/fundamentals/level03/OptionExercises1.scala

```diff
@@ -61,8 +70,16 @@ object OptionExercises1 {
     *
     * scala> mkPerson("Bob", -1)
     * = None
+    *
+    * You can solve this with standard if-statements, but let's try solve this using pattern matching.
+    * We create a pair `(name, age)` and pattern match on the values.
     **/
-  def mkPerson(name: String, age: Int): Option[Person] = ???
+  def mkPerson(name: String, age: Int): Option[Person] =
+    (name, age) match {
+      case (n, a) if n.isEmpty => ???
```

Comment:
Not sure about the use of guards here. Do you think it's better than using regular `if` statements?

Reply:
I thought this would be an opportunity to teach guards, but maybe we can put that aside.

I'll take this out. Otherwise it's just a once off and they won't remember it.